cellpack/autopack/loaders/config_loader.py

```diff
@@ -29,11 +29,13 @@ class ConfigLoader(object):
         "load_from_grid_file": False,
         "inner_grid_method": "trimesh",
         "live_packing": False,
+        "num_trials": 1,
         "name": "default",
         "ordered_packing": False,
         "out": "out/",
         "overwrite_place_method": False,
         "place_method": "jitter",
+        "rng_seed": None,
```

Comment:
can you make this a full word? I'm not sure what this is

Reply:
This is the random number generator (rng) seed. I can see that there could be ambiguity between interpreting this as a range of seeds as well. How about `randomness_seed`?

Reply:
yeah that seems good

Reply:
This has been updated